test/v2/connectors/AmendForeignPropertyPeriodSummaryConnectorSpec.scala

```diff
@@ -78,33 +77,57 @@ class AmendForeignPropertyPeriodSummaryConnectorSpec extends ConnectorSpec {
     foreignNonFhlProperty = Some(Seq(foreignNonFhlPropertyEntry))
   )
 
-  private val request: AmendForeignPropertyPeriodSummaryRequest = AmendForeignPropertyPeriodSummaryRequest(
-    nino = Nino(nino),
-    businessId = businessId,
-    taxYear = TaxYear.fromMtd(taxYear),
-    submissionId = submissionId,
-    body = requestBody
-  )
-
   trait Test {
     _: ConnectorTest =>
-    val connector: AmendForeignPropertyPeriodSummaryConnector = new AmendForeignPropertyPeriodSummaryConnector(
+    def taxYear: TaxYear
```

Comment:
Could Test be moved down into the end of the test class?  Similar to RetrieveForeignPropertyPeriodSummaryConnectorSpec  - makes the tests easier to read